File: Beecrowd-Problems/C#/done!/problem1008.cs
using System;
using System.Globalization;
class URI
{

    static void Main(string[] args)
    {
        CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
        CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
        int employeeNumber = int.Parse(Console.ReadLine());
        int amountPerHour = int.Parse(Console.ReadLine());
        double workedHours = double.Parse(Console.ReadLine());

        double salary = amountPerHour * workedHours;

        Console.WriteLine($"NUMBER = {employeeNumber}");
        Console.WriteLine($"SALARY = U$ {salary:F2}");
    }
}

File: Beecrowd-Problems/C++/done!/problem1008.cpp
#include <iostream>
#include <iomanip>

int main() {

    int employee_number, worked_hours;
    double salary;

    std::cin >> employee_number;
    std::cin >> worked_hours;
    std::cin >> salary;

    std::cout << std::fixed << std::setprecision(2);
    std::cout << "NUMBER = " << employee_number << std::endl;
    std::cout << "SALARY = U$ " << worked_hours * salary << std::endl;

    return 0;
}